src/graphql/schema/resolver-helpers.js

```diff
@@ -2,7 +2,8 @@ export const mergeProps = (key, ...objects) => ({
   key,
   ...objects.reduce((arr, obj) => ({ ...arr, ...(obj[key] || obj(key)) }), {})
 });
+export const users = (arrayOfUserIDs, User) => User.find({ _id: { $in: arrayOfUserIDs } });
 
-export const voters = ({ voters }, _, { User }) => voters.map((id) => User.getOne(id));
+export const voters = ({ voters }, _, { User }) => users(voters, User);
```

Comment:
that is not a correct DB-query, am I wrong?

Reply:
`users()` is a helper function that fetches the users based on an array of ids (here `voters`).